extensions-contrib/kafka-emitter/src/main/java/org/apache/druid/emitter/kafka/KafkaEmitter.java

```diff
@@ -166,6 +184,10 @@ public void emit(final Event event)
           if (!alertQueue.offer(objectContainer)) {
             alertLost.incrementAndGet();
           }
+        } else if (event instanceof RequestLogEvent && config.getRequestTopic() != null) {
```

Comment:
one question, why not still use `requestLost` as the counter for lost events instead of putting them in the bucket of `invalid` events which seems to be there for cases where the object isn't a supported event type?

Reply:
This is to keep the behaviour same as before where request log events were counted as `invalid`, now they will start seeing `requestLost` as 'N/A' in the logs if request topic is not set and the count in `invalidLost`.  But it may make more sense now to count them as `requestLost` as request log event is now supported.

Reply:
made that change